K2Bridge.Tests.UnitTests/Visitors/DateHistogramAggregationVisitorTests.cs

```diff
@@ -51,13 +51,13 @@ public string DateHistogramVisit_WithAggregation_ReturnsValidResponse(string int
             return histogramAggregation.KustoQL;
         }
 
-        [TestCase("w", ExpectedResult = "_data | summarize metric by ['key'] = startofweek(todatetime(['field'].['A']))\n| order by ['key'] asc", TestName = "DateHistogramVisit_WithStartOfWeekInterval_ReturnsValidResponse")]
-        [TestCase("week", ExpectedResult = "_data | summarize metric by ['key'] = startofweek(todatetime(['field'].['A']))\n| order by ['key'] asc", TestName = "DateHistogramVisit_WithStartOfWeekInterval_ReturnsValidResponse")]
-        [TestCase("y", ExpectedResult = "_data | summarize metric by ['key'] = startofyear(todatetime(['field'].['A']))\n| order by ['key'] asc", TestName = "DateHistogramVisit_WithStartOfYearInterval_ReturnsValidResponse")]
-        [TestCase("year", ExpectedResult = "_data | summarize metric by ['key'] = startofyear(todatetime(['field'].['A']))\n| order by ['key'] asc", TestName = "DateHistogramVisit_WithStartOfYearInterval_ReturnsValidResponse")]
-        [TestCase("M", ExpectedResult = "_data | summarize metric by ['key'] = startofmonth(todatetime(['field'].['A']))\n| order by ['key'] asc", TestName = "DateHistogramVisit_WithStartOfMonthInterval_ReturnsValidResponse")]
-        [TestCase("month", ExpectedResult = "_data | summarize metric by ['key'] = startofmonth(todatetime(['field'].['A']))\n| order by ['key'] asc", TestName = "DateHistogramVisit_WithStartOfMonthInterval_ReturnsValidResponse")]
-        [TestCase("z", ExpectedResult = "_data | summarize metric by ['key'] = bin(todatetime(['field'].['A']), z)\n| order by ['key'] asc")]
+        [TestCase("w", ExpectedResult = "\nlet _extdata = _data\n| extend ['key'] = startofweek(todatetime(['field'].['A']));\nlet _summarizablemetrics = _extdata\n| summarize metric by ['key']\n| order by ['key'] asc;", TestName = "DateHistogramVisit_WithStartOfWeekInterval_ReturnsValidResponse")]
```

Comment:
Do we need all of the newlines in the query? They're not meant to be readable anyway

Reply:
The 'newline' logic was really different from each individual Visitor. Some were using only simple command separator |, some were using KustoOperator.CommandSeparator (\n|) which includes newline. And I introduced KustoOperator.Newline (to be used only before let commands).

We can decide to remove all the \n if you agree. I don't have a real preference. The only situation where it helps, is copy/paste from the debugger. If I have your Go, I remove them.